tests/main/services-restart/task.yaml

```diff
@@ -61,7 +61,7 @@ execute: |
     test "$TIMESTAMP1_AFTER" -gt "$TIMESTAMP1_BEFORE"
     test "$TIMESTAMP2_AFTER" -gt "$TIMESTAMP2_BEFORE"
     test "$TIMESTAMP3_AFTER" -eq "$TIMESTAMP3_BEFORE"
-    test "$TIMESTAMP4_AFTER" -eq "$TIMESTAMP4_BEFORE"
+    test "$TIMESTAMP4_AFTER" -gt "$TIMESTAMP4_BEFORE"
```

Comment:
what's this change about, as there is little context from the diff alone

Reply:
Service 4 is in enabled and inactive state. Previously service 4 was not being restarted and now it is, so in the past the start time was not changing but now it does, to a later time.

Reply:
maybe a comment before all the tests,   "# greater means restarted"